lib/ttnt/testtask.rb

```diff
@@ -12,14 +13,21 @@ class TestTask
     # An instance of `Rake::TestTask` passed when TTNT::TestTask is initialized
     attr_accessor :rake_testtask
 
+    attr_accessor :test_files
+    attr_accessor :code_files
```

Comment:
This is great! Thank you!
Did it here: https://github.com/Genki-S/ttnt/commit/173fa9795b7ae67ed895900fbc7824b523d90428